int8 locatingGate			= 0
int8 aligningWithGate		= 1
int8 passingGate	    	= 2
int8 locatingPole	 		= 3
int8 approachingPole	   	= 4
int8 pivotingPole			= 5
